Warnings in Xcode 8.3 (Swift 3.1)

````
    - WARN  | xcodebuild:  ActionSheetPicker-3.0/Pickers/ActionSheetLocalePicker.h:46:24: warning: empty paragraph passed to '@param' command [-Wdocumentation]
    - WARN  | xcodebuild:  ActionSheetPicker-3.0/Pickers/ActionSheetLocalePicker.h:43:12: warning: parameter 'data' not found in the function declaration [-Wdocumentation]
    - WARN  | xcodebuild:  ActionSheetPicker-3.0/Pickers/ActionSheetMultipleStringPicker.h:45:24: warning: empty paragraph passed to '@param' command [-Wdocumentation]
    - WARN  | xcodebuild:  ActionSheetPicker-3.0/Pickers/ActionSheetMultipleStringPicker.h:43:12: warning: parameter 'index' not found in the function declaration [-Wdocumentation]
    - NOTE  | xcodebuild:  ActionSheetPicker-3.0/Pickers/ActionSheetMultipleStringPicker.h:43:12: note: did you mean 'indexes'?
    - WARN  | xcodebuild:  ActionSheetPicker-3.0/Pickers/ActionSheetStringPicker.h:42:24: warning: empty paragraph passed to '@param' command [-Wdocumentation]
````

--- a/Pickers/ActionSheetLocalePicker.h
+++ b/Pickers/ActionSheetLocalePicker.h
@@ -40,10 +40,9 @@ static const float secondColumnWidth = 160.0f;
  *  Create and display an action sheet picker.
  *
  *  @param title             Title label for picker
- *  @param data              is an array of strings to use for the picker's available selection choices
  *  @param index             is used to establish the initially selected row;
  *  @param target            must not be empty.  It should respond to "onSuccess" actions.
- *  @param successAction
+ *  @param successAction     successAction
  *  @param cancelActionOrNil cancelAction
  *  @param origin            must not be empty.  It can be either an originating container view or a UIBarButtonItem to use with a popover arrow.
  *

--- a/Pickers/ActionSheetMultipleStringPicker.h
+++ b/Pickers/ActionSheetMultipleStringPicker.h
@@ -40,9 +40,9 @@ typedef void(^ActionMultipleStringCancelBlock)(ActionSheetMultipleStringPicker *
  *
  *  @param title             Title label for picker
  *  @param data              is an array of strings to use for the picker's available selection choices
- *  @param index             is used to establish the initially selected row;
+ *  @param indexes           is used to establish the initially selected row;
  *  @param target            must not be empty.  It should respond to "onSuccess" actions.
- *  @param successAction
+ *  @param successAction     successAction
  *  @param cancelActionOrNil cancelAction
  *  @param origin            must not be empty.  It can be either an originating container view or a UIBarButtonItem to use with a popover arrow.
  *

--- a/Pickers/ActionSheetStringPicker.h
+++ b/Pickers/ActionSheetStringPicker.h
@@ -39,7 +39,7 @@ typedef void(^ActionStringCancelBlock)(ActionSheetStringPicker *picker);
  *  @param data              is an array of strings to use for the picker's available selection choices
  *  @param index             is used to establish the initially selected row;
  *  @param target            must not be empty.  It should respond to "onSuccess" actions.
- *  @param successAction
+ *  @param successAction     successAction
  *  @param cancelActionOrNil cancelAction
  *  @param origin            must not be empty.  It can be either an originating container view or a UIBarButtonItem to use with a popover arrow.
  *

